docs/reference/docs/bulk.asciidoc

```diff
@@ -738,3 +742,39 @@ The API returns the following result.
 }
 ----
 // TESTRESPONSE[s/"index_uuid": "aAsFqTI0Tc2W0LCWgPNrOA"/"index_uuid": $body.$_path/]
+
+
+[discrete]
+[[bulk-dynamic-templates]]
+===== Example with dynamic templates parameter
+
+The below example creates a dynamic template, then performs a bulk request 
+consisting of index/create requests with the `dynamic_templates` parameter.
+
+[source,console]
+----
+PUT my-index/
+{
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "geo_point": {
+         "mapping": {
+            "mapping" : "geo_point"
```

Comment:
is this syntax correct? should it be type: geo_point ? Can you also fix the indentation?

Reply:
Thanks, fixed in https://github.com/elastic/elasticsearch/pull/69948/commits/d0010bd1e3546caebf82af6a3e446b23964fedcb